src/lib.rs

```diff
@@ -446,16 +430,9 @@ impl<'a> ReportMessageBuilder<'a> {
     add_generic_field!(with_level, level, Into<Level>);
 
     // Send the message to Rollbar.
-    pub fn send(&mut self) -> thread::JoinHandle<Option<ResponseStatus>> {
+    pub async fn send(&mut self) -> Option<ResponseStatus> {
         let client = self.report_builder.client;
-
-        match self.report_builder.send_strategy {
-            Some(ref send_strategy) => {
-                let http_client = client.http_client.to_owned();
-                send_strategy(http_client, self.to_string())
-            }
-            None => client.send(self.to_string()),
-        }
+        client.send(self.to_string()).await
```

Comment:
this variation of send needed to stay async because it is called from a test? is the other send not called from a test? confused.